### PR TITLE
Move from Jest to uvu

### PR DESCRIPTION
**Pull request by usmanyunusov**
#### `Commit Message Summary (CHANGELOG)`

```
This PR replace Jest to UVU. 
```

### `Type`

- [x] CI
- [x] Fix
- [ ] Perf
- [ ] Docs
- [x] Test
- [ ] Chore
- [ ] Style
- [ ] Build
- [ ] Feature
- [ ] Refactor

### `SemVer`

- [x] Fix (:label: Patch)
- [ ] Feature (:label: Minor)
- [ ] Breaking Change (:label: Major)

### `Issues`

- Fixes `#235`

### `Checklist`

- [x] Lint and unit tests pass with my changes
- [x] I have added tests that prove my fix is effective/works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes are merged and published in downstream modules

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -53,12 +54,27 @@
       "optional": true
     }
   },
+  "c8": {
+    "exclude": [
+      "**/*.test.*"
+    ],
+    "lines": 100,
```
Comment (ai) [inline package.json:61]: Put limit to current `97.64`

File: package.json [hunk 2]
```diff
@@ -11,9 +11,8 @@
     "node": ">= 10"
   },
   "scripts": {
-    "lint": "standard --env jest",
```
Comment (ai) [inline package.json:14]: Removing `lint` was not part of task

File: test/utils.js [hunk 3]
```diff
@@ -1,17 +1,23 @@
...
+};
 
 const expected = (dir, file) => {
-  return fs.readFileSync(path.resolve('test', dir, 'expect', file), 'utf8')
-}
+  return fs.readFileSync(path.resolve("test", dir, "expect", file), "utf8");
+};
+
+const describe = (name, hook) => {
+  hook(suite(name));
```
Comment (ai) [inline test/utils.js:16]: This file code style if out of Prettier config

Comment (usmanyunusov) [reply test/utils.js]: Fixed with run `pnpm lint`

File: package.json (hunk 2 above)
Comment (usmanyunusov) [reply package.json:14]: Reverted

File: package.json (hunk 1 above)
Comment (usmanyunusov) [reply package.json]: Fixed

**Pull request merged**